client/src/reader/components/notes/FilteredList.js

```diff
@@ -67,16 +67,9 @@ class FilteredList extends PureComponent {
 
   renderHeading() {
     const { handleSeeAllClick, handleFilterChange } = this.props;
-    const { readingGroup, ...restFilters } = this.filters;
 
     return (
       <div className="notes-filtered-list__header">
-        <div className="notes-filtered-list__header-start">
-          <Partial.Filters
-            filterChangeHandler={handleFilterChange}
-            filters={restFilters}
```

Comment:
I didn't fully remove this component but can if we think we won't use it again.

Reply:
I think it's fine to leave it for now.